CHANGELOG.md

```diff
@@ -404,8 +412,9 @@ Install with `npm install onfido-sdk-ui@0.12.0-rc.1`
 
 
 [next-version]:
-https://github.com/onfido/onfido-sdk-ui/compare/5.0.1...development
-[5.0.1]: https://github.com/onfido/onfido-sdk-ui/compare/4.0.0...5.0.0
+[5.2.1]: https://github.com/onfido/onfido-sdk-ui/compare/5.1.0...5.2.1
```

Comment:
How did we go from `5.1.0` to `5.2.1`? Also the development link is missing

Reply:
5.2.0 got accidentally published on npm when making the release candidate. Npm doesn't allow to publish the same version again even tho I managed to unpublish it. That's why we had to bump patch.